packages/xchain-bitcoincash/README.md

```diff
@@ -9,3 +9,15 @@
 ```
 yarn add @xchainjs/xchain-bitcoincash
 ```
+
+## Service Providers
+
+This package uses the following service providers:
+
+| Function                    | Service        | Notes                                                 |
+| --------------------------- | -------------- | ----------------------------------------------------- |
+| Balances                    | Haskoin        | https://api.haskoin.com/#/Address/getBalance          |
+| Transaction history         | Haskoin        | https://api.haskoin.com/#/Address/getAddressTxsFull   |
+| Transaction details by hash | Haskoin        | https://api.haskoin.com/#/Transaction/getTransaction  |
+| Transaction broadcast       | Haskoin        | https://api.haskoin.com/#/Transaction/postTransaction |
+| Explorer                    | Blockchain.com | https://www.blockchain.com                            |
```

Comment:
If we don't have rate limits here, I would recommend to add following line (or something like that):
```
 Rate limits: No
 ```